python/benchmarks/plasma.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import numpy as np
+
+import pyarrow as pa
+import pyarrow.plasma as plasma
+
+from . import common
+
+
+class PlasmaBenchmarks(object):
+
+    def setup(self):
+        self.plasma_store_ctx = plasma.start_plasma_store(plasma_store_memory=10**9)
+        plasma_store_name, p = self.plasma_store_ctx.__enter__()
+        self.plasma_client = plasma.connect(plasma_store_name, "", 64)
+
+        self.data_1kb = np.random.randn(1000 // 8)
```

Comment:
You should use parametrization for the various sizes (you can look at the conversion benchmarks to see how that is done, or see [docs](https://asv.readthedocs.io/en/latest/writing_benchmarks.html#parameterized-benchmarks)). Also I don't think we should test so many sizes.  Testing a very small size (1kb) and a large-ish size (10mb) sounds sufficient.